srcpkgs/jump/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'jump'
+pkgname=jump
+version=0.30.1
+revision=1
+build_style=go
+go_import_path=github.com/gsamokovarov/jump
+go_package=github.com/gsamokovarov/jump/cmd/jump
+hostmakedepends="git"
+short_desc="Fuzzy directory jumper that learns your habits"
+maintainer="Arthur Weagel <arthur.contrib@gmail.com>"
+license="MIT"
+homepage="https://github.com/gsamokovarov/jump"
+changelog="https://github.com/gsamokovarov/jump/releases"
```

Comment:
Ideally something plaintext

Reply:
There is no `CHANGELOG` file, so this is the closest option I'm aware of (the author does provide some notes with each release). There's also the [atom feed version](https://github.com/gsamokovarov/jump/releases.atom) for the releases page if that's preferable.

Reply:
Better to not have it at all if it doesn't have it.

Reply:
Hmm, I see this being done in lots of package definitions (see [hub](https://github.com/void-linux/void-packages/blob/master/srcpkgs/hub/template), [erlang](https://github.com/void-linux/void-packages/blob/master/srcpkgs/erlang/template), [mpv](https://github.com/void-linux/void-packages/blob/master/srcpkgs/mpv/template), [obs](https://github.com/void-linux/void-packages/blob/master/srcpkgs/mpv/template), [skaffold](https://github.com/void-linux/void-packages/blob/master/srcpkgs/skaffold/template), [sysdig](https://github.com/void-linux/void-packages/blob/master/srcpkgs/sysdig/template), etc).

Are you sure it's better to remove it, or would it make sense to follow the existing convention? I don't have a strong preference.

Reply:
Well, that wasn't the intention, but if this is where it is, that's where it is. I will not object further.